src/preferences.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self'; script-src 'unsafe-inline'">
+  <link rel="stylesheet" href="./assets/css/bulma.min.css">
+  <link rel="stylesheet" href="./assets/css/style.css">
+  <title>Preferences</title>
+</head>
+
+<body>
+
+  <header>
+
+      <nav class="navbar" role="navigation" aria-label="main navigation">
+        <div class="navbar-brand">
+          <div class="navbar-item">
+            <img src="./icons/streaker-icon.svg" alt="Streaker Logo">
+          </div>
+          <div class="navbar-item">
+            <span>Streaker</span>
+          </div>
+        </div>
+      </nav>
+
+  </header>
+
+  <main>
+
+    <nav class="panel">
+      <p class="panel-heading">
```

Comment:
Could we use [labels](https://bulma.io/documentation/form/general/) instead of panel headings for these? Panel headings may be useful in the future when we have multiple options that sit in the same category.

Reply:
Is that what you wanted to do ?
![preview.png](https://i.imgur.com/dMVeJTt.png)

Reply:
I meant actual `<label>` elements, like this (name, username, email):
<img width="289" alt="screenshot 2018-10-25 at 08 38 32" src="https://user-images.githubusercontent.com/8133259/47483575-80c21400-d831-11e8-92b8-3bf00b70e515.png">

Reply:
Oh ok ! I will do that